src/lib/server/postgres_protocol_handler.hpp

```diff
@@ -7,6 +7,8 @@
 
 namespace opossum {
 
+typedef std::unordered_map<PostgresMessageType, std::string> ErrorMessage;
```

Comment:
[using is better than typedef](https://www.internalpointers.com/post/differences-between-using-and-typedef-modern-c)